### PR TITLE
Fix wrong function name in case of function in function

### PR DESCRIPTION
**Pull request by colin014**

### CODE REVIEW

File: formatter.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ const FileKey = "file"
 
 const (
 	logrusStackJump          = 5
```
Comment (bonifaido) [inline formatter.go]: I guess this constant has to be decreased as well :)

**Review by bonifaido — CHANGES_REQUESTED** [review]

File: formatter.go (hunk 1 above)
Comment (colin014) [reply formatter.go]: You're right :) 👍

**Review by bonifaido — APPROVED** [review]

**Comment by bonifaido** [comment]
LGTM, thank you @colin014  :)

**Review by matyix — APPROVED** [review]
thx @colin014

**Pull request merged**